package.json

```diff
@@ -68,7 +68,7 @@
 		"react-day-picker": "^6.2.1",
 		"react-motion": "^0.5.2",
 		"react-peek": "^1.0.5",
-		"react-transition-group": "^1.2.1",
+		"react-transition-group": "^4.4.1",
```

Comment:
This was the the upgrade. All other changes were a result of this migration to version 4.4.1